__tests__/app.test.js

```diff
@@ -406,3 +406,33 @@ describe('/api/articles/:article_id/comments', () => {
 		});
 	});
 });
+
+describe('/api/comments/:comment_id', () => {
+	describe('204: DELETE:', () => {
+		it('responds with status 204 and a message', () => {
+			return request(app).delete('/api/comments/1').expect(204);
+		});
```

Comment:
For extra confidence, you could also run another request in a .then here to make a request for the deleted comment and expect a 404 to come back